ambry-clustermap/src/test/java/com.github.ambry.clustermap/HelixParticipantTest.java

```diff
@@ -96,7 +97,6 @@ public void testSetReplicaSealedState() throws IOException {
     String instanceName = ClusterMapUtils.getInstanceName(hostname, port);
     HelixParticipant helixParticipant =
         new HelixParticipant(new ClusterMapConfig(new VerifiableProperties(props)), helixManagerFactory);
-    helixParticipant.initialize(hostname, port, Collections.EMPTY_LIST);
```

Comment:
Did you check if this test would fail without your diff if this call was removed?

Reply:
Yes. I have checked that. After removing `initialize()`, the unit test would fail without current changes.